requirements.txt

```diff
@@ -1 +1,4 @@
 requests~=2.31.0
+
+typing_extensions~=4.12.2
+PyGithub~=2.3.0
```

Comment:
```suggestion
requests~=2.31.0
typing_extensions~=4.12.2
PyGithub~=2.3.0

```

Reply:
Empty lines fixed in commit: [fb78190](https://github.com/AbsaOSS/living-doc-generator/pull/16/commits/fb78190e5adedc65ded39abff55adc6f51da5407).